public/sketch.js

```diff
@@ -1,7 +1,9 @@
 // Open and connect socket
-let socket = io();
+var socket = io();
 // Keep track of users
-let users = {};
+var users = {};
+// keep track of osc objects
+var osc = {};
```

Comment:
Be careful about making these `var` instead of `let`. It won't make much of a difference here though.